packages/ezsqlite/ezsqlite.0.4.2/opam

```diff
@@ -0,0 +1,20 @@
+opam-version: "2.0"
+maintainer: "Zach Shipko <zachshipko@gmail.com>"
+authors: ["Zach Shipko"]
+license: "ISC"
+homepage: "https://github.com/zshipko/ocaml-ezsqlite"
+bug-reports: "https://github.com/zshipko/ocaml-ezsqlite/issues"
+dev-repo: "git+https://github.com/zshipko/ocaml-ezsqlite.git"
+doc: "https://zshipko.github.io/ocaml-ezsqlite/doc"
+build: [
+    ["dune" "build" "-p" name "-j" jobs]
+]
+depends: [
+    "ocaml" {>= "4.03.0"}
+    "dune" {build}
+    "hex"
+]
+
+synopsis: """
+SQLite3 bindings
```

Comment:
Simple quotes should be enough for a synopsis. Also could you elaborate a little bit more? Your previous opam files had a longer and more complete `description` and `synopsis`.